pkg/mutator/testdata/release-v14.1.4.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v14.1.4
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.6.0
+  - name: chart-operator
+    version: 2.12.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 0.7.6
+    name: external-dns
+    version: 2.3.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.0
+  - name: metrics-server
+    version: 1.2.1
+  - name: net-exporter
+    version: 1.9.2
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.7.1
+  - name: cluster-autoscaler
+    version: 1.19.1
+  - name: azure-scheduled-events
+    version: 0.3.0
+  components:
+  - name: app-operator
+    releaseOperatorDeploy: true
+    version: 3.2.1
+  - name: azure-operator
+    releaseOperatorDeploy: true
+    version: 5.5.2
+  - name: cert-operator
+    reference: 0.1.0-2
+    releaseOperatorDeploy: true
+    version: 0.1.0
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.23.22
+  - name: kubernetes
+    version: 1.19.8
+  - name: containerlinux
+    version: 2605.12.0
+  - name: calico
+    version: 3.15.3
+  - name: etcd
+    version: 3.4.14
+  date: "2021-04-06T08:44:15Z"
+  state: active
+status:
+  inUse: false
+  ready: false
```

Comment:
Today, in nitpicking news: [Why should text files end with a newline?](https://stackoverflow.com/a/729795/563072)

Reply:
That was a nice read there :grin: